.github/workflows/deploy.yaml

```diff
@@ -151,6 +151,29 @@ jobs:
           openssl ec -in key.pem -pubout -out key.public.pem
           docker run -i danedmunds/pem-to-jwk:1.2.1 --public --pretty < key.public.pem > key.public.jwk
 
+      - name: 'Prepare GAIA-X self-description signature request'
+        uses: cuchi/jinja2-action@v1.2.0
+        with:
+          template: deployment/terraform/dataspace/WrappedParticipantSelfDescription.json.j2
+          output_file: deployment/terraform/dataspace/WrappedParticipantSelfDescription.json
+          strict: true
+          variables: |
+            registrationNumber=A999999
+            country=NL
+
+      - name: 'Request GAIA-X self-description signature'
+        run: |
+          curl --fail --location 'https://compliance.gaia-x.eu/api/v1/participant/signature/sign' \
+          --header 'Content-Type: application/json' \
+          --data @WrappedParticipantSelfDescription.json \
+          --output SignedWrappedParticipantSelfDescription.json
```

Comment:
Shouldn't this file be rather called something like `RegistryServiceSelfDescription.json`?

Reply:
For GX it seems the authority is also a participant. Nomenclature could use some clarification